### ISSUE TITLE
Installing the extension prevents from compiling any .qmd document

### ISSUE THREAD

**Issue by AV-BD** [shafayetShafee/downloadthis#4]
Thanks for the nice extension!
However I did not maage to make it work, I get this error mesage:

```Error running filter C:/Users/XXXXX/AppData/Local/Programs/Quarto/share/filters/main.lua:
[string "..."]:296: module 'base64.lua' not found:
        no field package.preload['base64.lua']
        no file 'N:\XXXXX\XXXXX\_extensions\shafayetShafee\downloadthis\resources/lua/base64.lua'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\lua\base64\lua.lua'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\lua\base64\lua\init.lua'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\base64\lua.lua'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\base64\lua\init.lua'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\..\share\lua\5.4\base64\lua.lua'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\..\share\lua\5.4\base64\lua\init.lua'       
        no file '.\base64\lua.lua'
        no file '.\base64\lua\init.lua'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\share\pandoc\datadir\base64\lua.lua'
        no file 'C:/Users/XXXXX/AppData/Local/Programs/Quarto/share/filters\base64\lua.lua'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\base64\lua.dll'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\..\lib\lua\5.4\base64\lua.dll'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\loadall.dll'
        no file '.\base64\lua.dll'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\base64.dll'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\..\lib\lua\5.4\base64.dll'
        no file 'C:\Users\XXXXX\AppData\Local\Programs\Quarto\bin\tools\loadall.dll'
        no file '.\base64.dll'
stack traceback:
        [string "..."]:296: in upvalue 'orig_require'
        [string "..."]:1353: in function 'require'
        (...tail calls...)
        ..._extensions\shafayetShafee\downloadthis\downloadthis.lua:53: in local 'chunk'
        ...rin/AppData/Local/Programs/Quarto/share/filters/main.lua:13966: in function 'initShortcodeHandlers'
        ...rin/AppData/Local/Programs/Quarto/share/filters/main.lua:15371: in main chunk
```

Looking forward for a solution, since it's supposed to dexactly what I am after!

**Comment by shafayetShafee** [comment]
What ia the quarto version that you are using?

**Comment by AV-BD** [comment]
Thank you for your quick reply!

I am using _1.3.321_ according to `quarto check`

**Comment by shafayetShafee** [comment]
Did you install the extension using the command `quarto add shafayetShafee/downloadthis`? ( I have checked this extension for versions `1.2.475`, `1.3.309` and `1.3.333`. It is working fine so far.

**Comment by shafayetShafee** [comment]
Anyway can you try installing the extension again using the command `quarto add shafayetShafee/downloadthis` and then try again? I have done some minor tweaks and released version `1.1.0`.
